etc/kayobe/kolla/globals.yml

```diff
@@ -9,8 +9,16 @@ kayobe_image_tags:
   openstack:
     rocky: zed-rocky-9-20230821T155947
     ubuntu: zed-ubuntu-jammy-20230821T155947
+  bifrost:
+    rocky: zed-rocky-9-20230927T142529
```

Comment:
Should we still have zed references? :thinking:

Reply:
There's not much point worrying about it either way, since we're always going to have different tags between versions. Once we build antelope containers we'll just overwrite these

Reply:
+1 - no antelope images yet